src/platform_clock.h

```diff
@@ -22,7 +22,10 @@
 #include <type_traits>
 #include <thread>
 
-#if defined(_3DS)
+#if defined(USE_LIBRETRO)
+#include "platform/libretro/libretro_clock.h"
+using Platform_Clock = LibretroClock;
```

Comment:
Just asking because the commit message does not show the intention clearly:
We want to always use the libretro clock, even if we have a "better" one for some platforms?

Reply:
That's a good question. I have to think about it

Reply:
I decided to only use the libretro clock where we use the "default clock"